packages/govuk-frontend/src/govuk/components/pagination/pagination.yaml

```diff
@@ -57,7 +61,11 @@ params:
       - name: text
         type: string
         required: false
-        description: The link text to the next page. Defaults to 'Next'.
+        description: The text content of the link to the next page. If `next.html` is supplied, this is ignored.
+      - name: html
+        type: string
+        required: false
+        description: The HTML content of the link to the next page. Defaults to 'Next page', with 'page' being visually hidden.
```

Comment:
```suggestion
        description: The HTML content of the link to the next page. Defaults to 'Next page', with 'page' being visually hidden. If `next.text` is supplied, this is ignored.
```

Reply:
Do we need to clarify that:

1. The `text` default is `'Previous page'`
2. The `html` default is `'Previous<span class="govuk-visually-hidden"> page</span>'`

Same for the **Next page** options

Reply:
I'm not sure if that's useful because it's not technically true, I guess?

If a user managed to override `html` in a way that it registered as being undefined but still bypassed the default (which, to be fair, I don't think is really possible with the current Nunjucks logic), there is no `text` default that would be fallen back upon. The only default is the one containing HTML.

Reply:
Not technically, it's hard to explain the behaviour isn't it 😣

For users that want to add `html` we should show the correct HTML "default" so to speak

But it's harder to justify `text` as yep there isn't a default

Reply:
I'm gonna keep the description of the default prosaic for now, rather than showing inline HTML, as that seems to be the more common way of describing these things (presumably if a person is comfy with code, they could just look at it already). Something else to hash out for the documentation style guide.